.github/test-infra/ci-iac-aws/zarf.yaml

```diff
@@ -0,0 +1,186 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/defenseunicorns/zarf/main/zarf.schema.json
+kind: ZarfPackageConfig
+metadata:
+  name: ci-iac-aws
+  description: "CI - IAC AWS (not for Prod use)"
+  # x-release-please-start-version
+  version: "0.15.1"
+  # x-release-please-end
+  architecture: amd64
+  source: https://github.com/defenseunicorns/uds-core
+  documentation: https://github.com/defenseunicorns/uds-core
+  vendor: Defense Unicorns
+
+variables:
+  - name: CLUSTER_NAME
```

Comment:
I dont think this should be a zarf package. It could be a couple of uds tasks called directly in CI. It would be much faster that way as well.

Reply:
^ ditto, I know this was probably path of least resistance for copy/paste from DUBBD but viewing now I think we can do this way simpler with tf/eksctl direct in a task. Would also handle multi-arch things without you having to think about it (currently terraform/eksctl for arm/mac are not in these packages).